### PR TITLE
Ignore tasks that are inside comment blocks

### LINKED ISSUE #10: Ignore task lists that are commented out
In our repo's PR template we have something like this:
```md
<!--
### ToDo
If there are any tasks that need to be done prior to this PR being merged, write them here.
A bot will make sure that you've checked them all before allowing merging the PR 
- [ ] task 1
-->
```

If you leave this as-is (i.e. commented out), the `task-list-completed` bot still takes them into account, and we have to manually remove it from the description.

The bot should **ignore** commented-out tasks.

### PR DESCRIPTION
**Pull request by bengry**
Fixes https://github.com/stilliard/github-task-list-completed/issues/10.

### CODE REVIEW

**Review by stilliard — APPROVED** [review]
Fantastic work!

**Comment by stilliard** [comment]
Hi @bengry 

Thanks for your work on this! It look great, switching to the markdown parsing makes perfect sense for this. 

Sorry for the delay looking into this btw, it's been busy lately but hoping to get into testing this during next week as well as introducing tests for it too :)

Will keep you updated, thank you

**Comment by bengry** [comment]
Thanks for looking into this. Waiting for your update on the merge.

@stilliard Somewhat unrelated - in the meantime I've set up a fork on Glitch that has this fix. However, I see they shut it down automatically after 5 minutes with no requests. How do you keep it running when not requests are made?

**Comment by stilliard** [comment]
@bengry I can't remember exactly but I think I might have set something up to ping the service regularly (maybe every minute or so) so it would know it's constantly in use :)

**Comment by bengry** [comment]
Thanks. I tried to use https://uptimerobot.com/, but it failed after a while.. Not sure why, and unfortunately I don't have the time to look into this.
Anyway, I've reverted to using the original app (and not my fork) in the meantime, and will wait for your merge.

**Comment by bengry** [comment]
@stilliard any news on merging this to `master` and publishing a new version of the app?

**Comment by stilliard** [comment]
Hey @bengry 
Thanks for the reminder, testing it now :)

**Comment by stilliard** [comment]
Awesome fix! Thanks for the contribution :+1:

**Comment by stilliard** [comment]
That's now all live & working

File: index.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+const marked = require('marked');
```
Comment (ali2030ksa1) [inline index.js:1]: شباب القصيم .,.,.,., سيرفر الحياة الواقعيه

**Pull request merged**